src/v/archival/service.cc

```diff
@@ -374,7 +338,7 @@ ss::future<> scheduler_service_impl::reconcile_archivers() {
     // find new ntps that present in the snapshot only
     for (const auto& [ntp, p] : pm.partitions()) {
         if (
-          ntp.ns != model::redpanda_ns && !_archivers.contains(ntp)
+          ntp.ns == model::kafka_namespace && !_archivers.contains(ntp)
```

Comment:
I guess they will be needed for full-cluster recovery, but it will require more work anyway.

BTW I am interested in justification too. Maybe factor this into a separate commit and write a justification in the commit message?